handler/src/main/java/io/netty/handler/ssl/JdkAlpnApplicationProtocolNegotiator.java

```diff
@@ -146,7 +145,10 @@ public SSLEngine wrapSslEngine(SSLEngine engine, ByteBufAllocator alloc,
     }
 
     static boolean jdkAlpnSupported() {
-        return PlatformDependent.javaVersion() >= 9 && Java9SslUtils.supportsAlpn();
+        // While it is called Java9SslUtils it is not strictly true anymore that this only exists in Java9+ as it
+        // was recently backported as part of https://bugs.java.com/bugdatabase/view_bug.do?bug_id=8230977.
+        // Because of this lets not do a Java version runtime check but just depend on if the methods are present.
+        return Java9SslUtils.supportsAlpn();
```

Comment:
If `Java9SslUtils` is no longer related to Java 9, why not renaming it (it's package protected).

Reply:
@slandelle I was not able to find a better name yet :D

Reply:
Me neither...

Reply:
@slandelle `AlpnSslEngine` ?

Reply:
`AlpnSslEngine` and `AlpnSslUtils` LGTM